msg/benches/req_res_localhost.rs

```diff
@@ -0,0 +1,81 @@
+use bytes::Bytes;
+use futures::StreamExt;
+use rand::Rng;
+
+use msg_socket::{RepSocket, ReqSocket};
+use msg_transport::Tcp;
+
+static N_REQS: usize = 100000;
+static PAR_FACTOR: usize = 64;
+static REPEAT_TIMES: usize = 10;
+
+/// Benchmark the throughput of a single request/response socket pair over localhost
+fn main() {
+    tracing_subscriber::fmt::init();
```

Comment:
If you change this to `let _ = tracing_subscriber::fmt::try_init()` it won't panic when there's already a subscriber registered